src/elements/content-uploader/ContentUploader.js

```diff
@@ -959,13 +961,16 @@ class ContentUploader extends Component<Props, State> {
      * @return {void}
      */
     handleUploadProgress = (item: UploadItem, event: any) => {
+        const { onProgress } = this.props;
```

Comment:
This can be used to just above its first use, similar to the `this.state` destructuring in this same method.

Reply:
👍